### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by RadekNowak2k19**

### CODE REVIEW

**Review by NerdbordMentor — CHANGES_REQUESTED** [review]
## Code Review Report

### Task Goal
Update **readme.md** file, and correct the wrong sentence.

### Requirements
1. Typo should be fixed in **readme.md** file.

### Code Changes Review

The code changes in the **readme.md** file are as follows:

```diff
@@ -1,15 +1,13 @@
 # html-css-js-workout
+
 Coding challenges designed to test your HTML, CSS, and JavaScript skills.
-As you complete each challenge, you'll be able to submit your code for review and feedback. 
+As you complete each challenge, you'll be able to submit your code for review and feedback.
 
 ## Instructions
 
-- open ``index.html``
+- open `index.html`
 - each task that you do in repo should be a separate pull request
 
 ### **Are you here to complete the task?**
 
-`I do nto sea eny typo!`
-
-
-
+`I do not sea any typo!`
```

The changes made in the file partially meet the requirements of the task. The typo in the sentence `I do nto sea eny typo!` has been corrected to `I do not sea any typo!`. However, there is still a typo in the word "sea" which should be "see". Therefore, the sentence should be corrected to `I do not see any typo!`.

In conclusion, the changes made do not fully cover the task requirements as there is still a typo present in the sentence.
